.github/ISSUE_TEMPLATE/dependabot.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Dependabot / Dependency Upgrades
+about: Issue template to account for effort spent on dependency upgrades in a sprint
+title: ''
+labels: engineering
```

Comment:
```suggestion
labels: engineering, dependabot
```
Not sure if this is correct syntax for multiple labels, but adding a dependabot label automatically would be useful.